cli/src/main.rs

```diff
@@ -320,27 +319,30 @@ fn run() -> Result<(), CliError> {
                     .value_name("application_metadata")
                     .takes_value(true)
                     .multiple(true)
-                    .help("Application metadata for the circuit proposal"),
+                    .help("Application metadata of the proposal"),
             )
             .arg(
                 Arg::with_name("metadata_encoding")
                     .long("metadata-encoding")
                     .takes_value(true)
                     .possible_values(&["json", "string"])
                     .requires("metadata")
-                    .help("Set the encoding for the application metadata. Default value: string"),
+                    .help(
+                        "Set encoding of application metadata \
+                           Default value: string",
```

Comment:
Can this line be put in parentheses or brackets? Currently it displays as `Set encoding of application metadata Default value: string [possible values: json, string]` Maybe `... metadata (default value: string) [possible values: ...]` would look best?

Reply:
Took out `value` just to cut down on characters but went with your formatting suggestion